tests/test-sources/modules/output.nix

```diff
@@ -1,7 +1,92 @@
+{ pkgs, helpers, ... }:
+let
+  inherit (pkgs) lib;
+in
```

Comment:
I think we can remove this whole section and access these (where needed) via module args in each test case's module

E.g.

```nix
all-configs.module = 
  {
    config,
    pkgs,
    lib,
    helpers,
    ...
  }:
```

Reply:
I wanted to ask something unrelated. Is it intended that I can't just use module as a function without `module` attr? It's because here

https://github.com/nix-community/nixvim/blob/6dc0bda459bcfb2a38cf7b6ed1d6a5d6a8105f00/tests/default.nix#L47

it doesn't handled that it can be a function. To workaround this I have to do `all-configs.module` instead of just `all-configs`

Reply:
> Is it intended that I can't just use module as a function without module attr?

It's not explicitly intended, it's just an edge-case I decided not to handle in order to keep the implementation simpler.

We could gate the `removeAttrs` behind a `isAttrs` or `? tests` check, but doing so felt a little bloated when I initially added full module support to the tests. :shrug: 

Don't really mind either way.